packages/monaco/src/browser/monaco-command.ts

```diff
@@ -150,3 +213,50 @@ export class MonacoEditorCommandHandlers implements CommandContribution {
     }
 
 }
+
+class EditorQuickOpenItem extends QuickOpenItem {
+
+    constructor(
+        private readonly type: string,
+        private readonly label: string,
+        protected readonly execute: (type: string) => void) {
```

Comment:
I just wonder if it wouldn't be preferable to extend `QuickOpenItemOptions` and add the options you add in that interface? I think label does the thing as the original options.label no?